tests/unit/test_exceptions.py

```diff
@@ -1,42 +1,36 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later
 # pylint: disable=missing-module-docstring
 
+from parameterized import parameterized
 from tests import SearxTestCase
 import searx.exceptions
 from searx import get_setting
 
 
 class TestExceptions(SearxTestCase):  # pylint: disable=missing-class-docstring
-    def test_default_suspend_time(self):
-        with self.assertRaises(searx.exceptions.SearxEngineAccessDeniedException) as e:
-            raise searx.exceptions.SearxEngineAccessDeniedException()
+    @parameterized.expand(
+        [
+            searx.exceptions.SearxEngineAccessDeniedException,
+            searx.exceptions.SearxEngineCaptchaException,
+            searx.exceptions.SearxEngineTooManyRequestsException,
+        ]
+    )
+    def test_default_suspend_time(self, exception):
+        with self.assertRaises(exception) as e:
+            raise exception()
         self.assertEqual(
             e.exception.suspended_time,
-            get_setting(searx.exceptions.SearxEngineAccessDeniedException.SUSPEND_TIME_SETTING),
+            get_setting(exception.SUSPEND_TIME_SETTING),
         )
 
-        with self.assertRaises(searx.exceptions.SearxEngineCaptchaException) as e:
-            raise searx.exceptions.SearxEngineCaptchaException()
-        self.assertEqual(
-            e.exception.suspended_time, get_setting(searx.exceptions.SearxEngineCaptchaException.SUSPEND_TIME_SETTING)
-        )
-
-        with self.assertRaises(searx.exceptions.SearxEngineTooManyRequestsException) as e:
-            raise searx.exceptions.SearxEngineTooManyRequestsException()
-        self.assertEqual(
-            e.exception.suspended_time,
-            get_setting(searx.exceptions.SearxEngineTooManyRequestsException.SUSPEND_TIME_SETTING),
-        )
-
-    def test_custom_suspend_time(self):
-        with self.assertRaises(searx.exceptions.SearxEngineAccessDeniedException) as e:
-            raise searx.exceptions.SearxEngineAccessDeniedException(suspended_time=1337)
+    @parameterized.expand(
+        [
+            searx.exceptions.SearxEngineAccessDeniedException,
+            searx.exceptions.SearxEngineCaptchaException,
+            searx.exceptions.SearxEngineTooManyRequestsException,
+        ]
+    )
+    def test_custom_suspend_time(self, exception):
+        with self.assertRaises(exception) as e:
+            raise exception(suspended_time=1337)
```

Comment:
I don't think the actual number is important in this test, so I didn't bother trying for the other random values